semgrep/Makefile

```diff
@@ -12,3 +12,12 @@ install:
 
 .PHONY: qa-test
 qa-test: PYTHONPATH=. pytest tests/ --qa
+
+.PHONY: benchmark
+benchmark:
+	PYTHONPATH=. pytest tests/ --benchmark-only $(ARGS)
+
+.PHONY: benchmark-snapshot
+benchmark-snapshot:
+	@read -p "Name your snapshot: " snapshot_name;\
+	PYTHONPATH=. pytest tests/ --benchmark-only --benchmark-save=$$snapshot_name $(ARGS)
```

Comment:
Can we use `--benchmark-autosave` instead so we don't have to request user input? Having to repeatedly enter in snapshot names as you're benchmarking may get tedious, and won't work in CI environments.

Reply:
Good call. Updated!